huaweicloud/services/css/resource_huaweicloud_css_thesaurus.go

```diff
@@ -0,0 +1,225 @@
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
+
+		Schema: map[string]*schema.Schema{
+			"region": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"cluster_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"bucket_name": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"main_object": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"stop_object": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"synonym_object": {
+				Type:     schema.TypeString,
+				Optional: true,
+			},
+			"status": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"update_time": {
+				Type:     schema.TypeInt,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func ResourceCssthesaurusCreate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+	cssV1Client, err := config.CssV1Client(region)
+	if err != nil {
+		return diag.Errorf("Error creating CSS V1 client: %s", err)
+	}
+	opts := buildThesaurusCreateParameters(d)
+	clusterId := d.Get("cluster_id").(string)
+
+	loadErr := thesaurus.Load(cssV1Client, clusterId, *opts)
+	if loadErr.Err != nil {
+		return diag.Errorf("load thesaurus to css cluster failed. cluster_id=%s,error=%s", clusterId, loadErr.Err)
+	}
+
+	createResultErr := checkThesaurusLoadResult(ctx, cssV1Client, clusterId, d.Timeout(schema.TimeoutCreate))
+	if createResultErr != nil {
+		return diag.FromErr(createResultErr)
+	}
+
+	d.SetId(clusterId)
+
+	return ResourceCssthesaurusRead(ctx, d, meta)
+}
+
+func buildThesaurusCreateParameters(d *schema.ResourceData) *thesaurus.LoadThesaurusReq {
+	opts := thesaurus.LoadThesaurusReq{
+		BucketName: d.Get("bucket_name").(string),
+	}
+
+	if obj, ok := d.GetOk("main_object"); ok {
+		opts.MainObject = obj.(string)
+	}
+	if obj, ok := d.GetOk("stop_object"); ok {
+		opts.StopObject = obj.(string)
+	}
+	if obj, ok := d.GetOk("synonym_object"); ok {
+		opts.SynonymObject = obj.(string)
+	}
+
+	return &opts
+}
+
+func ResourceCssthesaurusRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	config := meta.(*config.Config)
+	region := config.GetRegion(d)
+	cssV1Client, err := config.CssV1Client(region)
+	if err != nil {
+		return diag.Errorf("Error creating CSS V1 client: %s", err)
+	}
+
+	detail, err := thesaurus.Get(cssV1Client, d.Id())
+	if err != nil {
+		return diag.Errorf("Query cluster thesaurus failed,cluster_id=%s,err=%s", d.Id(), err)
+	}
+
+	mErr := multierror.Append(
+		d.Set("cluster_id", detail.ClusterId),
+		d.Set("bucket_name", detail.Bucket),
+		d.Set("main_object", detail.MainObj),
+		d.Set("stop_object", detail.StopObj),
+		d.Set("stop_object", detail.StopObj),
+		d.Set("synonym_object", detail.SynonymObj),
+		d.Set("status", detail.Status),
+		d.Set("update_time", detail.UpdateTime),
```

Comment:
`update_time` is a timestamp, not friendly, pls convert to "2006-01-02 15:04:05 MST" format